conf/xml/RobotStateProvider_iCub3.xml

```diff
@@ -37,7 +37,7 @@
         sparseCholeskyDecomposition,
         robustCholeskyDecomposition,
         sparseRobustCholeskyDecomposition -->
-        <param name='inverseVelocityKinematicsSolver'>sparseCholeskyDecomposition</param>
+        <param name='inverseVelocityKinematicsSolver'>QP</param>
```

Comment:
Are you sure that this option was working with iCub3 model, and without `CUSTOM_CONSTRAINTS` ? 
(related issue https://github.com/robotology/human-dynamics-estimation/issues/167)